mage_ai/tests/streaming/sinks/test_generic_io.py

```diff
@@ -52,33 +52,40 @@
 
 
 class GenericIOTests(TestCase):
-    @patch('mage_ai.streaming.sinks.generic_io.ConfigFileLoader')
+    def setUp(self):
+        super().setUp()
+        self.test_path = Path('./test')
```

Comment:
Let's replace `./test` with self.repo_path

Reply:
Done making the suggested change.